spec/entry_spec.rb

```diff
@@ -54,10 +54,28 @@
     it 'can handle multiple locales' do
       vcr('entry_locales') {
         cat = create_client.entries(locale: "*").items.first
-        expect(cat.fields[:name][:'en-US']).to eq "Nyan Cat"
-        expect(cat.fields[:name][:'es']).to eq "Gato Nyan"
+        expect(cat.fields('en-US')[:name]).to eq "Nyan Cat"
+        expect(cat.fields('es')[:name]).to eq "Gato Nyan"
```

Comment:
I'm not quite sure why we are reversing the order of parameters here? It'll be a breaking change.

In any case, the locale parameter should be a symbol, otherwise it'll be inconsistent with `field_with_locales`.